lib/manager/components/UserAccountInfoPanel.js

```diff
@@ -38,7 +38,7 @@ export default class UserAccountInfoPanel extends Component<Props> {
             </Col>
             <Col md={8}>
               <h4 style={{marginTop: 0, marginBottom: 15}}>
-                {this.messages('hello')}, {profile.nickname}.
```

Comment:
Yep!

Reply:
I think it would be best to use placeholders within a single string.

Reply:
I think Datatools i18n doesn't support placeholders. Elsewhere in the code we manually implement placeholders: 
<img width="580" alt="image" src="https://user-images.githubusercontent.com/63798641/235483117-ddc569cd-9140-4aa0-b4cc-f00ff9a47f73.png">

For this one liner I'm not sure this would make sense.